Package.swift

```diff
@@ -4,27 +4,55 @@
 import PackageDescription
 
 let package = Package(
-    name: "SwiftGraphQL",
+    name: "swift-graphql",
```

Comment:
This means users will have to specify `.product` in their targets dependencies.
Error:
<img width="672" alt="image" src="https://user-images.githubusercontent.com/17001151/106726631-e6093000-660a-11eb-9f84-5aa377afcd01.png">
Fix: 
<img width="552" alt="image" src="https://user-images.githubusercontent.com/17001151/106726774-0df89380-660b-11eb-884c-c431a4806547.png">

Is this intended? In my opinion it's easier to just specify the name in the package dependencies like users had to do previously:

```swift
.package(name: "SwiftGraphQL", url: "https://github.com/maticzav/swift-graphql", .branch("next")),
//       ^^^^^^^^^^^^^^^^^^^^
```

Or am I misunderstanding?

Reply:
I agree that this is a bit cumbersome, but it separates the code in the library much more clearly. I think we'll expose some additional tools that we use to generate code and it makes sense to enforce such split. I don't think this is a problem since it only takes about three sentences to explain it in the docs.